src/handlers/tracehandler.ts

```diff
@@ -47,10 +47,11 @@ export class TraceHandler {
   handleTrace(block_num, block_timestamp, trace) {
     const [api_message, trx] = trace;
 
-    if (api_message !== 'transaction_trace_v0') return;
-    this.stats.add('txs');
-    for (const action of trx.action_traces) {
-      this.handleAction(block_num, block_timestamp, trx, action);
+    if (api_message === 'transaction_trace_v0') {
```

Comment:
Shouldn't this be the same lines 101 and 102?

Reply:
https://github.com/AntelopeIO/leap/releases/tag/v3.1.0
looks like the changes are only with action_trace_v0  and chain_config_v0